### ISSUE TITLE
Tag release, publish to pypi?

### ISSUE THREAD

**Issue by digdugg** [11craft/louie#6]
With the merge of the python 3 commits and passing Travis CI builds maybe it's time to update pypi repo?  This way people can leverage pip to get the py3 friendly Louie

**Comment by gldnspud** [comment]
Agreed. I was going to do a few things before publishing:

[ ] contributors file
[ ] publish docs to readthedocs
[ ] test/fix it against pypy (there's an old PR for this that fell by the wayside)

**Comment by gldnspud** [comment]
OK, those three things are done now. We should take a look at https://packaging.python.org/en/latest/distributing/ and do any other prep work. I can do that early next week. @digdugg let me know if you'd like to do any of that, and if you have any other cleanup suggestions.

(I'll assume we can just make this version 1.1.1 since we are not making API or functionality changes, just Python 3 and PyPy compatibility, and removing some non-code related cruft.)

**Comment by digdugg** [comment]
Only other suggestions would be to link to Django signals library for possible usages of this library.  I think one of the commenters on the original repo were asking for usage examples.  The link is here and is well documented https://docs.djangoproject.com/es/1.9/topics/signals/

If I get time I can try to pitch in but don't foresee having much free time in the near term.

**Comment by sirex** [comment]
When will this be released to pypi?

**Comment by timcoote** [comment]
Bit of a hiatus here. pypi version seems to be from 2006 and doesn't support python3 :-(

**Comment by guy881** [comment]
Guys, could you please release python 3 compatible version to PyPI? I am migrating a tool which is using this library.

**Comment by gldnspud** [comment]
Sorry this took so long!

I bumped the version to 2.0, which now requires Python 3.6+, and released it to PyPI: https://pypi.org/project/Louie/

Still need to fix the readthedocs build, will keep this issue open until https://louie.readthedocs.io/en/latest/ is updated.

**Comment by matthewryanscott** [comment]
@proto3 @guy881 @timcoote @sirex @digdugg I [released 2.0.1 to PyPI](https://pypi.org/project/Louie/2.0.1/) just now.

I don't often use Louie myself, and the original co-author hasn't been involved in many years. If any of you are interested in helping with maintenance of this library, please reach out to me, and I can give you commit access and we can briefly chat about the library.

```
$ pip install louie
Collecting louie
  Downloading Louie-2.0.1-py3-none-any.whl (17 kB)
Installing collected packages: louie
Successfully installed louie-2.0.1

$ python
Python 3.11.4 (main, Jul 12 2023, 00:57:04) [Clang 14.0.3 (clang-1403.0.22.14.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import louie
>>> def receiver(*args, **kwargs):
...     print(f"{args=} {kwargs=}")
...     return 42
...
>>> louie.connect(receiver, "signal")
>>> louie.send("signal", "abc", number=123)
args=() kwargs={'signal': 'signal', 'sender': 'abc', 'number': 123}
[(<function receiver at 0x104c27060>, 42)]
```
